beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -1,7 +1,7 @@
 odoo.define('beescoop.pos', function (require) {
```

Comment:
OCA convention is : module_name.type (type = screens / models / etc...)

In our case : `odoo.define('beesdoo_pos.screens', function (require) {...})`